tests/nested/manual/muinstaller-real/task.yaml

```diff
@@ -58,6 +58,8 @@ execute: |
   # build updated shim
   version=22
   snap download --basename=pc --channel="$version/edge" pc
+  cp pc.snap pc.snap.orig
+  snap ack pc.assert
```

Comment:
Is this snap ack needed for any reason? Afaiu this does not affect the nested system.

Reply:
The fakestore code needs it, when missing it will error with:
```
$ fakestore make-refreshable --dir /home/gopath/src/github.com/snapcore/snapd/tests/nested/manual/muinstaller-real/fake-store-blobdir --snap-orig-blob pc.snap.orig --snap-blob ./pc-gadget pc
error: copying asserts: snap-revision (GDAw4TcTkCLpdnVLb5jTG9NOEoMFbXmimgjv42UpAi_fEO1Yi9jRg_XFG5LU6mwc;) not found
```
maybe there is a more elegant way but I don't know it :/

Reply:
Fwiw, I added a comment locally now and will push together with the function improvement suggested above

Reply:
Thanks for adding the comment